docs/catalog/ai/leverage-sustainable-regions.md

```diff
@@ -0,0 +1,39 @@
+---
+version: 1.0
+submitted_by: navveenb
+published_date: tbd
+category: ai
+tags: 
+ - ai
+ - machine-learning
+ - role:data-scientist
+ - size:small
+---
+
+# Use sustainable regions for ML training 
+
+## Description
+Training an AI model has a significant carbon footprint. Leverage
+a region which offers carbon-free energy for running your ML training jobs.
```

Comment:
Remove the "leverage a region" part as it is duplicated in the solution section. I would also suggest expanding on why training an AI model produces a significant carbon footprint (high compute, increased embodied carbon, etc)

Reply:
Done. Thanks